agent/internal/client/interface.go

```diff
@@ -48,8 +48,8 @@ type Communicator interface {
 	// GetExpansions returns all expansions for the task known by the app server
 	GetExpansions(context.Context, TaskData) (util.Expansions, error)
 	// Heartbeat sends a heartbeat to the API server. The server can respond with
-	// an "abort" response. This function returns true if the agent should abort.
-	Heartbeat(context.Context, TaskData) (bool, error)
+	// an "abort" response. Returning evergreen.TaskFailed signals the agent to abort the task.
```

Comment:
Same question as above.

Reply:
If the comment is going to be the same across the interface and the 2 implementations (+ mock), would it be easier to just stick the comment above the Heartbeat interface method and remove all the duplicate comments? I'm assuming the implementations will all have the same behavior.

Reply:
Yes they will - removed comments on the implementations to make the comment easier to maintain